src/utils/dialog/dialog.js

```diff
@@ -186,6 +189,38 @@ cwc.utils.Dialog.prototype.showTemplate = function(title, template, values) {
 
 
 /**
```

Comment:
Sorry for not providing the full context of this, I will work on an implementation which works directly over the soy templates.
So for now its fine to have it in src/ui/gcloud/gcloud.js and after I implemented this in the soy templates, I will adjust your current implementation as well.